### PR TITLE
Tutorial updated to latest version of kn

### LINKED ISSUE #281: Script install-istio.sh doesn't work for minikube 1.23.2
**Describe the bug**
Can't install istio on minikube 1.23 via the script `install-istio.sh.`
Here is the releases:

    $ minikube version
    minikube version: v1.23.2
    commit: 0a0ad764652082477c00d51d2475284b5d39ceed
    
    $ kubectl version
    Client Version: version.Info{Major:"1", Minor:"22", GitVersion:"v1.22.2",   GitCommit:"8b5a19147530eaac9476b0ab82980b4088bbc1b2", GitTreeState:"clean", BuildDate:"2021-09-15T21:38:50Z", GoVersion:"go1.16.8", Compiler:"gc", Platform:"linux/amd64"}
    Server Version: version.Info{Major:"1", Minor:"22", GitVersion:"v1.22.2", GitCommit:"8b5a19147530eaac9476b0ab82980b4088bbc1b2", GitTreeState:"clean", BuildDate:"2021-09-15T21:32:41Z", GoVersion:"go1.16.8", Compiler:"gc", Platform:"linux/amd64"}

    $ kubectl api-resources --api-group=apiextensions.k8s.io -o wide
    NAME                        SHORTNAMES   APIVERSION                NAMESPACED   KIND                       VERBS
    customresourcedefinitions   crd,crds     apiextensions.k8s.io/v1   false        CustomResourceDefinition   [create delete deletecollection get list patch update watch]

**To Reproduce**
1. Run `$BOOK_HOME/bin/install-istio.sh`
2. See error:

    unable to recognize "install/kubernetes/helm/istio-init/files/crd-10.yaml": no matches for kind "CustomResourceDefinition" in version "apiextensions.k8s.io/v1beta1"

**Expected behavior**
The script runs without error and istio gets installed

### PR DESCRIPTION
**Pull request by ammbra**

### CODE REVIEW

File: bin/start-minikube.sh [hunk 1]
```diff
@@ -12,16 +12,16 @@ if [ "$unamestr" == "Darwin" ];
 then
   minikube start -p "$PROFILE_NAME" \
   --memory="$MEMORY" \
-  --driver=hyperkit \
+  --driver=virtualbox \
```
Comment (sebastienblanc) [inline bin/start-minikube.sh]: Any reason to not use `hyperkit` ? For instance I use `hyperkit` on my mac to solve network issue with my minikube.

File: documentation/modules/advanced/pages/camel-k-cbr.adoc [hunk 2]
```diff
@@ -203,126 +180,183 @@ Since the fruits API returns a static set of fruit data consistently, you can ca
...
+                - to: "log:high?showAll=true&multiline=true"
 ----
 
 <1> The Camel route connects to Apache Kafka broker and the topic `fruits`.
 <2> Once the data is received it is transformed into a JSON payload.
 <3> The `Content Based Router` pattern using the https://camel.apache.org/manual/latest/choice-eip.html[Choice EIP]. In the data processing you classify the fruits as low (sugar pass:[&lt;=] 5), medium(sugar between 5 to 10) and high(sugar > 10) based on the sugar level present in their nutritions data.
 <4> Based on the data classification you will be setting the https://cloudevents.io[CloudEvents] `type` header to be `low-high`, `medium-sugar` and `high-sugar`. This header is used as one of the filter attributes in the Knative Eventing Trigger.
-<5> The last step is to send the processed data to the Knative Eventing Broker named `default`.
+
+Let's deplot the Kamelet using the following command:
```
Comment (sebastienblanc) [inline documentation/modules/advanced/pages/camel-k-cbr.adoc]: deploy

File: documentation/modules/camelk/pages/camel-k-eventing.adoc [hunk 3]
```diff
@@ -17,62 +17,44 @@ Navigate to the tutorial chapter's folder {camelk-repo}:
 cd $TUTORIAL_HOME/{camelk-repo}
 ----
 
-[#deploy-camel-k-source]
-== Deploy Knative Eventing CamelSource
+[#install-camel-k-plugin]
+== Install the kn kamelet source plugin
 
-The `CamelSource` allows you use a Camel K integration as part of the Knative Eventing architecture. Simply speaking, you can make the Camel K integration act as a https://knative.dev/docs/eventing/sources/[Knative Event Source] and  send the Camel exchanges (OUT) through a Knative Event Sink.
+The `Kamelet` allows you use a Camel K integration as part of the Knative Eventing architecture. Simply speaking, you can make the Camel K integration act as a https://knative.dev/docs/eventing/sources/[Knative Event Source] and  send the Camel exchanges (OUT) through a Knative Event Sink.
```
Comment (sebastienblanc) [inline documentation/modules/camelk/pages/camel-k-eventing.adoc]: "allows you to use" ?

**Review by sebastienblanc — COMMENTED** [review]
Impressive PR ! Good stuff !
One thing we need to check (after merging this one) is to get rid of the hardcoded namespace so that we can use this tutorial for the sandbox as well.

File: bin/start-minikube.sh (hunk 1 above)
Comment (ammbra) [reply bin/start-minikube.sh]: I had Virtualbox. Will switch to hiperkit (also locally) :)

File: documentation/modules/camelk/pages/camel-k-eventing.adoc (hunk 3 above)
Comment (ammbra) [reply documentation/modules/camelk/pages/camel-k-eventing.adoc]: Will rephrase that paragraph. I just re-read it and I do not like it that much :D looks confusing

**Review by lordofthejars — APPROVED** [review]

**Pull request merged**